lua/pantran/curl.lua

```diff
@@ -21,6 +26,11 @@ function curl:_spawn(request, path, data, callback)
     tostring(self._url / path)
   }
 
+  if self.config.proxy.enable then
+    table.insert(args, 1, ("%s:%s"):format(self.config.proxy.host, self.config.proxy.port))
```

Comment:
According to the curl manual specifying the port is optional, but this would fail here. I think it would be better to expect the entire proxy configuration to be a single string in the format `[protocol://]host[:port]` and thus leave the formatting to the user.